release/version.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
Wasn't aware. I just took this more or less from the kubernetes version logic. But I'll switch to your example.